apps/nextra/pages/en/build/sdks/ts-sdk/legacy-ts-sdk/sdk-tests.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: "Tests and Validation"
+slug: "typescript-sdk-tests"
```

Comment:
hmm... since this doesn't match, can we rename the file to typescript-sdk-tests and drop the slug?